scalalib/test/src/HelloWorldTests.scala

```diff
@@ -847,7 +847,6 @@ object HelloWorldTests extends TestSuite {
     'typeLevel - workspaceTest(HelloWorldTypeLevel){ eval =>
       val classPathsToCheck = Seq(
         HelloWorldTypeLevel.foo.runClasspath,
-        HelloWorldTypeLevel.foo.ammoniteReplClasspath,
```

Comment:
This check breaks because `HelloWorldTypeLevel` is a scala 2.11 ScalaModule and support for 2.11 has been dropped in Ammonite 1.6.9

Reply:
Without knowing what this line does, can we disable it only in case we run under Scala 2.11?

Reply:
Instead of ignoring the problem, I surfaced the ammonite version as a target in the scalalib trait : f386c83 

This will allow people still building against 2.11 to run ammonite on their projects via the `repl` command